airflow/dags/python_test.py

```diff
@@ -0,0 +1,69 @@
+"""
+Code that goes along with the Airflow located at:
+http://airflow.readthedocs.org/en/latest/tutorial.html
+"""
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from datetime import datetime, timedelta
+from airflow.operators.python_operator import PythonOperator
+
+import hashlib
+import json
+import pathlib
+from functools import partial
+from multiprocessing import Pool, cpu_count
+import requests
+
+def enrich_policy_watch():
+    with open(pathlib.Path('resources') / 'covid19db.json', encoding='utf-8') as covid19db:
+        covid19json = json.loads(covid19db.read())
+
+    list_count = len(covid19json)
+    current_item = 0
+
+    for field_data in covid19json:
+        current_item += 1
+        print('[' + str(current_item) + ' / ' + str(list_count) + '] - ' + field_data['fieldData']['title'])
+        # map.pool has SERIOUS LOCK/DEADLOCK ISSUES ! Remove the code and make it sequential !
```

Comment:
This code seems to explain and offer a solution to the problem. 
[article on multi threading](https://pythonspeed.com/articles/python-multiprocessing/)

This probably means that the multi-threaded code (parallel downloader) shall live in a separate module, and get only called when needed.

Reply:
perhaps, in general, it is a good idea to keep the functionality and the DAG definition in separate modules